src/backends/plonky2/mainpod/mod.rs

```diff
@@ -347,12 +347,14 @@ impl PodProver for Prover {
     }
 }
 
+pub type MainPodProof = ProofWithPublicInputs<F, C, D>;
+
 #[derive(Clone, Debug)]
 pub struct MainPod {
     params: Params,
     id: PodId,
     public_statements: Vec<Statement>,
-    proof: ProofWithPublicInputs<F, C, D>,
+    proof: MainPodProof,
```

Comment:
This change is mostly so that it's possible to import `MainPodProof` from the front-end without having to import `F`, `C` and `D`, which feel like they are implementation details that the front-end should not know about. Happy to be corrected on this! Also I see that there's a merge conflict for this line, so possibly this type needs to change.

Reply:
The alias looks good to me!
As for the conflict, it should be easy to resolve, you should update the alias to be:
```rust
pub type MainPodProof = Proof<F, C, D>;
```